sdks/jreleaser-git-java-sdk/src/main/java/org/jreleaser/sdk/git/ChangelogGenerator.java

```diff
@@ -608,6 +612,11 @@ Map<String, Object> asContext(boolean links, String commitsUrl) {
             context.put("commitTitle", passThrough(title));
             context.put("commitAuthor", passThrough(author.name));
             context.put("commitBody", passThrough(body));
+            context.put("commitIssuesCount", issues.size() > 0 ? issues.size() : null);
+            context.put("commitIssues", issues.stream().map(i -> {
+                String issue = links ? passThrough("[#" + i + "](" + issueTrackerUrl + i + ")") : "#" + i;
+                return Collections.singletonMap("issue", issue);
```

Comment:
Prefer the use of static imports

Reply:
would that be for `Collections.singletonMap` ?

Reply:
Yes, sorry. I should had been more explicit.